frontend/awx/views/schedules/hooks/scheduleHelpers.tsx

```diff
@@ -77,6 +77,48 @@ export function useGetSchedulCreateUrl(sublistEndPoint?: string) {
   return createUrl;
 }
 
+export function useGetSchedulEditUrl(sublistEndPoint?: string) {
+  const getPageUrl = useGetPageUrl();
+  const params = useParams<{
+    inventory_type?: string;
+    id: string;
+    source_id: string;
+    schedule_id?: string;
```

Comment:
schedule_id should not be optional here
```suggestion
    schedule_id: string;
```